examples/multi-cluster-fleet/python/agent_sandbox_fleet/__init__.py

```
@@ -0,0 +1,57 @@
+    # Client-side
+    "AssignmentsMissingError",
+    "ClusterResolver",
+    "ClusterUnavailableError",
+    "FleetSandboxClient",
+    "ResolvedCluster",
+    "ResolverError",
+    "TemplateNotAssignedError",
+    "gke_context_naming",
+    "kind_context_naming",
+    "resolve_cluster",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)